ThunderCloud/BundleDiagnosticsTableViewController.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  BundleDiagnosticsTableViewController.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 08/04/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import UIKit
+import ThunderTable
+
+class BundleDiagnosticTableViewController: TableViewController {
+    
+    let dateFormatter = DateFormatter()
+    
+    override func viewDidLoad() {
+        
+        super.viewDidLoad()
+        
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.dateFormat = "EEEE, MMM d, yyyy HH:mm Z"
+        
+        navigationController?.navigationBar.prefersLargeTitles = false
+        title = "Bundle Information"
+        
+        redraw()
+    }
+    
+    // MARK: - Redrawing
+    
+    private func timestampOfBundle(in directory: URL?) -> TimeInterval? {
+        guard let directory = directory else { return nil }
+        let manifestURL = directory.appendingPathComponent("manifest.json")
+        guard let jsonObject = try? JSONSerialization.jsonObject(with: manifestURL) else {
```

Comment:
Personal preference would be to  use `Codeable` and `JsonDecoder` and throw on our errors, then the caller can silence the error if they want? This still works good though!

Reply:
We okay to leave this as-is for now @BenCShutt ? :D

Reply:
Yeah absolutely, still works fine!